main.rb

```diff
@@ -0,0 +1,8 @@
+require './app'
+
+def main
+  puts 'Welcome to The School Library:'
+  app = List.new
+  app.option_list
+end
+main
```

Comment:
- [ ] `required`
- [ ] Amazing work so far!! 👏 
- [ ] When I run the `Ruby main.rb` I'm unable to choose any option because it exiting
- [ ] please not that its suppose to work like this [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)
- [ ] Please make it work 🥂